device-types/Cisco/N7K-C7009.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Cisco
+model: N7K-C7009
+slug: n7k-c7009
```

Comment:
The slug should have `cisco-` prepended to it

Reply:
The slug has been prepended with "cisco-".